client/deployments/client.go

```diff
@@ -49,15 +88,92 @@ func NewClient(url string, skipVerify bool) *Client {
 
 	return &Client{
 		url:               url,
-		artifactUploadUrl: JoinURL(url, artifactUploadUrl),
+		artifactUploadURL: JoinURL(url, artifactUploadURL),
+		artifactsListURL:  JoinURL(url, artifactsListURL),
 		client: &http.Client{
 			Transport: tr,
 		},
 	}
 }
 
-func (c *Client) UploadArtifact(description, artifactPath, tokenPath string, noProgress bool) error {
+func (c *Client) ListArtifacts(tokenPath string, detailLevel int) error {
+	token, err := ioutil.ReadFile(tokenPath)
+	if err != nil {
+		return errors.Wrap(err, "Please Login first")
+	}
+
+	req, err := http.NewRequest(http.MethodGet, c.artifactsListURL, nil)
+	if err != nil {
+		return errors.Wrap(err, "Cannot create request")
+	}
+	req.Header.Set("Authorization", "Bearer "+string(token))
+
+	reqDump, _ := httputil.DumpRequest(req, false)
+	log.Verbf("sending request: \n%v", string(reqDump))
```

Comment:
```suggestion
	log.Verbf("sending request: \n%s", string(reqDump))
```

Reply:
Also interesting that it has `Verbf`, but not `Infof`

Reply:
OK